language-plutus-core/test/Evaluation/Machines/Budget/Fib/3.plc.golden

```diff
@@ -2,7 +2,7 @@
 , ({ tally: ({ BBuiltin (StaticStagedBuiltinName SubtractInteger) causes ({ cpu: 4
 | mem: 4
 })
-| BBuiltin (StaticStagedBuiltinName AddInteger) causes ({ cpu: 2
+| BBuiltin (StaticStagedBuiltinName AddInteger) causes ({ cpu: 0
 | mem: 2
```

Comment:
Was `cpu: 2, mem: 2` and now `cpu: 0, mem: 2`?

Reply:
I never said the model was already calibrated properly :-)